bindings/go/nvml/nvml.go

```diff
@@ -128,6 +128,35 @@ func (t ProcessType) String() string {
 	return typ
 }
 
+type BrandType uint
+
+const (
+	BrandUnknown BrandType = iota
+	BrandQuadro
+	BrandTesla
+	BrandNvs
+	BrandGrid
+	BrandGeforce
+)
+
+func (t BrandType) String() string {
+	switch t {
+	case BrandUnknown:
+		return "Unknown"
+	case BrandQuadro:
+		return "Quadro"
+	case BrandTesla:
+		return "Tesla"
+	case BrandNvs:
+		return "Nvs"
+	case BrandGrid:
+		return "Grid"
+	case BrandGeforce:
+		return "Geforce"
+	}
```

Comment:
Can you reproduce this: https://github.com/NVIDIA/libnvidia-container/blob/master/src/driver.c#L651-L687

Reply:
Sounds good to you?